src/_tests_/logic/calculate.test.js

```diff
@@ -0,0 +1,18 @@
+import calculate from '../../logic/calculate';
```

Comment:
- [ ] Based on the requirements of this project, all the functionalities, and scenarios of the calculate.js module should be covered using jest. Hence let's kindly consider the scenarios where the button name(buttonName) is equal to zero('0'), not zero('!0'), dot(.), plus/minus('+/-').

Reply:
Thank you for the review.